lcls-twincat-common-components/Library/Devices/PPM/FB_PPM_PowerMeter.TcPOU

```diff
@@ -77,10 +85,71 @@ VAR
     '}
     fCalibMJRatio: LREAL;
 
+    // Linked to beam rate
+    {attribute 'pytmc' := '
+        pv: BeamRateFromEpics
+        link: TPG:SYS0:1:DST04:RATE
```

Comment:
Is there some variability here depending on which accelerator is in use? Do we need to have some special handling for this?